pywbem/cim_operations.py

```diff
@@ -1127,6 +1169,20 @@ def last_reply_len(self):
         """
         return self._last_reply_len
 
+    @property
+    def conn_id(self):
+        """
+        :term:`unicode string`:
+        String that is attached to each log entry output to uniquely identify
+        the WBEMConnection responsible for that log.  This is also part of
+        the log name for the operation and xml logs so that logging could
+        be enabled differently for different wbem connections. It is created
+        when the WBEMConnection object is created and remains constant
+        thorughout the life of that connection. I is unique for each
+        connection.
```

Comment:
I suggest to have a first sentence that just describes what it identifies and how unique it is, before we go into describing how it is used in logging (because that is just one possible use).

Reply:
Also, we should introduce a term such as "connection id" because we are referencing that term in the logging documentation.